### PR TITLE
Add com.orama_interactive.Pixelorama

### PR DESCRIPTION
**Pull request by Calinou**
Follow-up to #1299.

[Pixelorama](https://github.com/OverloadedOrama/Pixelorama) is an open source sprite editor based on Godot Engine.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.orama_interactive.Pixelorama

File: com.orama_interactive.Pixelorama.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      - type: file
+        url: https://github.com/Orama-Interactive/Pixelorama/raw/45868536369e3df4651681df8ccfb4141d05cbcb/Misc/Linux/com.orama_interactive.Pixelorama.appdata.xml
+        sha256: 39e7d79431e697bf9d20854447ea92c43ddfba3d072927dd61c40047cc41dc64
+
+    build-commands:
+      - install -Dm755 pixelorama.sh /app/bin/pixelorama
+      - install -Dm644 Pixelorama.pck /app/bin/pixelorama.pck
+      - mv Brushes/ /app/bin/Brushes/
+      - install -Dm644 icon.png -t /app/share/icons/hicolor/256x256/apps/
```
Comment (bilelmoussaoui) [inline com.orama_interactive.Pixelorama.yaml]: Why not just installing the icon with the correct name and removing the `rename-icon`?

Comment (Calinou) [reply com.orama_interactive.Pixelorama.yaml]: I guess both ways are equivalent, it's just that the current way doesn't require spelling out the app ID yet another time :slightly_smiling_face: 

Still, I can change this if you want.

Comment (bilelmoussaoui) [reply com.orama_interactive.Pixelorama.yaml]: Just use `${FLATPAK_ID}`

Comment (Calinou) [reply com.orama_interactive.Pixelorama.yaml]: I didn't know about this variable, thanks :slightly_smiling_face:

That said, the `.desktop` file references an icon named `pixelorama`, so I'd have to use `desktop-file-edit` manually. In contrast, `rename-icon` does this for me automatically, which is less error-prone.

Comment (bilelmoussaoui) [reply com.orama_interactive.Pixelorama.yaml]: Right, makes sense. Anyway, that's not a blocker, but those files should be renamed correctly upstream as the naming conventions are not Flatpak specific.

File: com.orama_interactive.Pixelorama.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+runtime-version: "19.08"
+default-branch: stable
+sdk: org.freedesktop.Sdk
+command: pixelorama
+rename-icon: icon
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=host
```
Comment (bilelmoussaoui) [inline com.orama_interactive.Pixelorama.yaml]: Just use home filesystem access here, these permissions can be easily changed later. 
Other than this, it should be ready

Comment (Calinou) [reply com.orama_interactive.Pixelorama.yaml]: That makes sense, my only concern is that it removes the ability to edit files present on removable media (if I read the docs correctly). It's probably not a huge issue for most people, but people may wonder why they can't browse removable media.

**Comment by bilelmoussaoui** [comment]
bot, build com.orama_interactive.Pixelorama

**Comment by bilelmoussaoui** [comment]
Let us know if anyone else from upstream needs write access to the repository

**Comment by OverloadedOrama** [comment]
@bilelmoussaoui I'd like write access to the repository!

**Comment by barthalion** [comment]
@OverloadedOrama Invite sent, make sure to accept it.

**Pull request closed without merging**